frontend/src/bundles/goals/pages/goals.tsx

```diff
@@ -104,107 +109,115 @@ const Goals: React.FC = () => {
     const lastGoal = goals.filter((goal) => goal.completedAt !== null).at(-1);
+    const goalType = lastGoal
+        ? activityToGoal[lastGoal.activityType]
+        : GoalTypes.STANDART;
 
     return (
-        <main className="bg-secondary ml-auto mr-auto flex w-full max-w-[71rem] flex-col gap-8 xl:flex-row xl:justify-normal">
+        <main className="bg-secondary flex w-full flex-col justify-center gap-8 xl:flex-row">
             {isLoading ? (
                 <Loader isOverflow />
             ) : (
                 <>
-                    <div className="flex w-full flex-col gap-8 xl:w-[70%]">
-                        <section>
-                            <GoalWidget
-                                value={
-                                    (lastGoal?.distance as number) ||
-                                    (lastGoal?.duration as number)
-                                }
-                                target={
-                                    (lastGoal?.distance as number) ||
-                                    (lastGoal?.duration as number)
-                                }
-                                title={
-                                    lastGoal
-                                        ? GOALS_MESSAGES.GOAL_COMPLETED
-                                        : GOALS_MESSAGES.NO_GOALS
-                                }
-                                subTitle={
-                                    lastGoal
-                                        ? GOALS_MESSAGES.GOAL_ENCOURAGE
-                                        : ''
-                                }
-                                goalType={
-                                    lastGoal
-                                        ? activityToGoal[lastGoal.activityType]
-                                        : GoalTypes.STANDART
-                                }
-                                hasAchievement={Boolean(lastGoal)}
-                                hasDistance={Boolean(lastGoal?.distance)}
-                            />
-                        </section>
+                    {!isSubscribed && (
+                        <GoogleAds className="hidden h-[63rem] max-w-64 flex-1 2xl:flex 2xl:text-[15px]" />
+                    )}
+                    <div className="flex max-w-[1136px] flex-col gap-8 xl:flex-row">
+                        <div className="flex w-full flex-col gap-8 xl:w-[70%]">
+                            <section>
+                                <GoalWidget
+                                    value={
+                                        (lastGoal?.distance as number) ||
+                                        (lastGoal?.duration as number)
+                                    }
+                                    target={
+                                        (lastGoal?.distance as number) ||
+                                        (lastGoal?.duration as number)
```

Comment:
You can write it once and store in variable. 
`const YOUR_VARIABLE_NAME= (lastGoal?.distance as number) || (lastGoal?.duration as number);`

```
value={YOUR_VARIABLE_NAME}
target={YOUR_VARIABLE_NAME}
```